### PR TITLE
Improve make_format error message.

### PR DESCRIPTION
**Pull request by briantkennedy**
Adjust output to print what was wrong and resolution.

### CODE REVIEW

File: scripts/check_format.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+diff="$(opa fmt -d lib/ validator/)"
+
+if [[ "$diff" == "" ]]; then
```
Comment (AdrienWalkowiak) [inline scripts/check_format.sh]: I would rather check the return code for the previous command ($? -ne 0)

File: scripts/check_format.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  exit
+fi
+
+echo "Formating error:"
+echo "$diff"
+echo ""
+echo "Run \"make format\" to fix."
```
Comment (AdrienWalkowiak) [inline scripts/check_format.sh]: nit: suggest saying run make format locally to reproduce this error

File: scripts/check_format.sh (hunk 1 above)
Comment (briantkennedy) [reply scripts/check_format.sh]: it exits 0 diff or no diff.

File: scripts/check_format.sh (hunk 2 above)
Comment (briantkennedy) [reply scripts/check_format.sh]: fixed

File: scripts/check_format.sh (hunk 1 above)
Comment (AdrienWalkowiak) [reply scripts/check_format.sh]: I believe you can redirect the error to the standard output:

diff="$(echo opa fmt -d lib/ validator/ 2>&1)"

if [[ $? -ne 0 ]];
then

Comment (briantkennedy) [reply scripts/check_format.sh]: If there is a diff, it exits 0.  If there is no diff, it exits 0.  We can't rely on the exit code for whether or not there is a diff.

Comment (AdrienWalkowiak) [reply scripts/check_format.sh]: sounds good. it's too bad opa does not behave in a typical way

**Pull request merged**